src/components/contact-form.svelte

```diff
@@ -228,7 +222,7 @@
       on:blur={() => dispatch('blur', 'message')}
       on:input={() => dispatch('input', 'message')}
     />
-    {#if type === 'quote'}
+    {#if type === 'quote' && budgetOptions}
```

Comment:
is this check increment needed?
```suggestion
    {#if type === 'quote'}
```